packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -947,24 +948,66 @@ abstract class ResidentRunner {
   void printHelp({ @required bool details });
 
   void printHelpDetails() {
+    if (flutterDevices.any((FlutterDevice d) => d.device.supportsScreenshot)) {
+      printCommandHelp(
+          's', 'Save a screenshot to flutter.png.');
+    }
     if (supportsServiceProtocol) {
-      printStatus('You can dump the widget hierarchy of the app (debugDumpApp) by pressing "w".');
-      printStatus('To dump the rendering tree of the app (debugDumpRenderTree), press "t".');
+      printCommandHelp('w', 'Dump widget hierarchy to the console.',
+          inParenthesis: 'debugDumpApp');
+      printCommandHelp('t', 'Dump rendering tree to the console.',
+          inParenthesis: 'debugDumpRenderTree');
       if (isRunningDebug) {
-        printStatus('For layers (debugDumpLayerTree), use "L"; for accessibility (debugDumpSemantics), use "S" (for traversal order) or "U" (for inverse hit test order).');
-        printStatus('To toggle the widget inspector (WidgetsApp.showWidgetInspectorOverride), press "i".');
-        printStatus('To toggle the display of construction lines (debugPaintSizeEnabled), press "p".');
-        printStatus('To simulate different operating systems, (defaultTargetPlatform), press "o".');
-        printStatus('To toggle the elevation checker, press "z".');
+        printCommandHelp('L', 'Dump layer tree to the console.',
+            inParenthesis: 'debugDumpLayerTree');
+        printCommandHelp('S', 'Dump accessibility tree in traversal order.',
+            inParenthesis: 'debugDumpSemantics');
+        printCommandHelp(
+            'U', 'Dump accessibility tree in inverse hit test order.',
+            inParenthesis: 'debugDumpSemantics');
```

Comment:
For cases like this where putting it all on one line isn't going to work, this is the preferred style:

```dart
        printCommandHelp(
￼          'U', 'Dump accessibility tree in inverse hit test order.',
￼          inParenthesis: 'debugDumpSemantics',
        );
```

dartfmt will use this style (or something close to it) if you put a trailing comma on the arguments list.